wp-content/themes/core/components/blocks/gallery_grid/css/dialog.pcss

```diff
@@ -0,0 +1,268 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Component: Dialog
+ *
+ * ----------------------------------------------------------------------------- */
+
+.c-dialog {
+
+    .swiper-wrapper {
+        align-items: center;
+        padding: 20px 0;
```

Comment:
Please use the existing spacer values for any padding/margins (here and throughout).